packages/framework-integration-tests/src/config/config.ts

```diff
@@ -25,6 +25,12 @@ function configureInvocationsHandler(config: BoosterConfig) {
   }
 }
 
+function configureBoosterSensorHealth(config: BoosterConfig) {
+  Object.values(config.sensorConfiguration.health.booster).forEach((indicator) => {
+    indicator.enabled = true
```

Comment:
Why is this needed? Does`config.sensorConfiguration.health.booster` refer to Booster default health checks? If so, Is the reason for this loop that those default health checks are disabled by default?

If so, Does a normal application need to do this to enable Booster internal health checks?

(so many questions! 😂 )

Reply:
hahaha, np, thanks for your questions and suggestions, all of them makes sense. Let me try to answer to all of them:

1. Why is this needed: By default all the health checks are disabled by default. We could add a method to enable all the defaults at once. Thoughts?
2. Doesconfig.sensorConfiguration.health.booster refer to Booster default health checks?: Yes
3. Is the reason for this loop that those default health checks are disabled by default?: Yes
4. Does a normal application need to do this to enable Booster internal health checks?: Yes

Reply:
@gonzalogarciajaubert Thanks for answering. Regarding point 1. First a question: Are the booster health sensor referred by name so that we can enable just some of them? I mean, can we do something like:
```typescript
const sensors = config.sensorConfiguration.health.booster
sensors.readmodels.enabled = true
sensors.eventStore.enabled = true
// etc
```
If so, I think we don't need that method, because it is always good to have that explicitly in the configuration file. If not, then it could be hard for the user to enable some of them and not others, as they would need to do an... array search? In that case, I would provide some methods to facilitate this or store the sensors with a name, as written above

Reply:
You can do:

```typescript
  const sensors = config.sensorConfiguration.health.booster
  sensors[BOOSTER_HEALTH_INDICATORS_IDS.DATABASE].enabled = true
```

Reply:
@gonzalojaubert That's enough. Could you please add an example like this in the docs?